qiskit/transpiler/passes/routing/bip_mapping.py

```diff
@@ -84,9 +84,9 @@ def __init__(
                 If None, all qubits in the coupling_map will be considered.
             objective (str): Type of objective function:
 
-                * ``'error_rate'``: [NotImplemented] Predicted error rate of the circuit
+                * ``'error_rate'``: Predicted error rate of the circuit
```

Comment:
can you add some description of how this "prediction" works? I think it multiplies gate errors, and thus it is only a function of circuit size but not depth. But just needs to be explicit.

Reply:
Add more description on 'error_rate' objective at 0c24c75

Reply:
I think `error_rate` is not a good name because a true error rate is a combination of gate errors and depth of circuit. This completely ignores depth. We should just call it `size`. So we have "size"/"depth"/"balanced"

Reply:
I agree "error_rate" is not a precise name and "size" is better, but "size" still sounds like a bit misleading name to me since the objective considers 2q-gate errors (not counts). How about "2qgate_error"? I think it's the most appropriate name. Is it too long?

Reply:
good point. If we want to be completely accurate in naming these objective function then even depth might not be too accurate. I'll merge this for now and if we find a better name we can replace.

Reply:
changed to "gate_error" for now.